### ISSUE TITLE
do not mark "only if"

### ISSUE THREAD

**Issue by rriemann** [btford/write-good#92]
Currently, the word "only" in "only if" is marked. I think this is a legitimate use and should not be marked.

**Comment by RichardLitt** [comment]
I'm not sure only should be marked at all. It's a conjunction most of the time, not an adverb.

**Comment by brandonkal** [comment]
Yes, "only" should not be considered a weasel word. It's often necessary for normal english statements without a comparable synonym.

**Comment by btford** [comment]
I'm open to removing it.

**Comment by Mister-Hope** [comment]
2020 10 19.  Agree only is a necessary word.
